matrix_sdk/src/room_member.rs

```diff
@@ -53,24 +54,15 @@ impl RoomMember {
     ///     .unwrap();
     /// let members = room.members().await.unwrap();
     /// let member = members.first().unwrap();
-    /// if let Some(avatar) = member.avatar(Some(96), Some(96)).await.unwrap() {
+    /// if let Some(avatar) = member.avatar(MediaFormat::File).await.unwrap() {
     ///     std::fs::write("avatar.png", avatar);
     /// }
     /// # })
     /// ```
-    pub async fn avatar(&self, width: Option<u32>, height: Option<u32>) -> Result<Option<Vec<u8>>> {
```

Comment:
why is the width-height option removed here?

Reply:
it's included in `MediaFormat` https://matrix-org.github.io/matrix-rust-sdk/matrix_sdk/media/enum.MediaFormat.html